docs/console/tutorials/project-infrastructure/project-configuration.md

```diff
@@ -0,0 +1,396 @@
+  # IMPORTANT: Replace with a strong, unique password. DO NOT commit this in plain text.
+  # Consider using environment variables or a secrets manager.
+  password           = "<STRONG_PASSWORD>"
+  project_id         = mongodbatlas_project.demo_project.id
+  roles {
+    role_name       = "readWrite"
+    database_name   = "admin" # Consider limiting to a specific database for production
+  }
+  auth_database_name = "admin" # The authentication database for the user
+  labels = {
+    env = "demo"
+  }
+}
+
+```
+
+## Deploying the Infrastructure Resource
+
+![IMG 7](./img/7.png)
+
+From the Deploy section, you can deploy infrastructure resources.
+
+Inside, you can see a table listing the created infrastructure resources, with their name and git repository.
+
+The "Start Plan" button performs a plan operation on the resource, which analyzes the terraform code in the repository and shows what would happen if the described changes were applied.
+
+The plan essentially informs what will be created, modified, or destroyed in case of applying the resource.
+
+![IMG 8](./img/8.png)
+
+Once the plan is executed, its result will show the changes that will occur if the resource is applied.
+
+If the plan is successful, the resource can be deployed by clicking the "Apply" button.
+
+## Example of GitLab CI Pipeline to Deploy the Resources
+
+Of course, the GitLab CI pipeline is essential for deploying the resources. In the following sections, we will provide an example of a GitLab CI pipeline that can be used to deploy the resources.
+
+:::info
+These are example snippets. Always adapt them to your specific requirements, thoroughly validate all settings (especially CI/CD variables and secrets), and remember that many effective solutions can be designed differently.
+:::
+
+```yaml
+
+# GitLab CI/CD Terraform Pipeline: Your Lean Starting Point
+#
+# Hey there! This pipeline is your foundation for automating Terraform with GitLab.
+# We're covering the essentials: validating your code, planning changes, and
+# applying them safely.
+#
+# Before you dive in, a few key things you'll need to tailor:
+#
+# 1. Terraform Version: Find `image: hashicorp/terraform:1.8.1` and set your team's version.
+# 2. Cloud Provider Authentication: This is crucial! Look for the `CRITICAL: Cloud Provider Authentication`
+#    section in `before_script`. You'll need to add your specific cloud login steps there.
+# 3. .terraformrc (Optional): If you use a custom `.terraformrc` for things like private provider
+#    registries, you can load it using the `TF_CONFIG_RC_CONTENT` CI/CD variable.
+# 4. Target Workspace: We use `TF_WORKSPACE` (defaulting to "default"). Set this as a CI/CD
+#    variable in GitLab to switch between environments like 'dev', 'staging', or 'production'.
+
+workflow:
+  rules:
+    - if: $CI_COMMIT_BRANCH
+    - if: $CI_MERGE_REQUEST_ID
+
+default:
+  image: hashicorp/terraform:1.8.1 # <-- SET YOUR PREFERRED TERRAFORM VERSION HERE
```

Comment:
I'd make an example with opentofu here to make sure we don't run in any licensing dispute